### PR TITLE
feat: support export of LoggingLogMetric

### PR DESCRIPTION
**Pull request by justinsb**
Wiring up support for our direct controllers here.

### CODE REVIEW

File: pkg/controller/direct/registry.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+			}
+			found, err := a.Find(ctx)
+			if err != nil {
+				return nil, err
+			}
+			if !found {
+				return nil, fmt.Errorf("resource %q is not found", url)
+			}
+
+			u, err := a.Export(ctx)
```
Comment (acpana) [inline pkg/controller/direct/registry.go:78]: Should I rename this `Export` to avoid any confusion in the future?

Comment (justinsb) [reply pkg/controller/direct/registry.go:78]: I think this `Export` is well-named.  It also makes sense to have a package-level `Export` function, which delegates to a particular Adapter's `Export`

**Comment by acpana** [comment]
/lgtm
/approve

thanks for this 🥇 

i'll rebase/ rework https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1691 to leverage this 💯

**Pull request merged**